core/object.cpp

```diff
@@ -2123,21 +2123,38 @@ void ObjectDB::cleanup() {
 	if (slot_count > 0) {
 		spin_lock.lock();
 
-		WARN_PRINT("ObjectDB Instances still exist!");
```

Comment:
I'd take the opportunity to make the "Instances" lower case (same in the other warning).

Reply:
Good point. Fixed. I've also inverted the `if` so it's clearer that both messages need to be updated if one is changed. I thought about doing it in a more elegant way, but it didn't pay.